collection-scripts/gather_nodes

```diff
@@ -104,8 +99,15 @@ do
         oc cp $pod:/host/etc/pcidp/config.json $NODE_PATH/pcidp_config.json -n node-gather 2>/dev/null
     fi
 
+}
+
+for line in $(oc get pod -o=custom-columns=NODE:.spec.nodeName,NAME:.metadata.name --no-headers --field-selector=status.phase=Running -n node-gather)
+do
+    gather_single_pod "$line" &
 done
 
+wait
```

Comment:
wouldn't it add unnecessary clutter to the console ? 
`wait 2>/dev/null` should hide it

Reply:
Only in interactive shell - http://git.savannah.gnu.org/cgit/bash.git/tree/jobs.c#n3022